mysql-ballerina/client.bal

```diff
@@ -151,7 +151,7 @@ type ClientConfiguration record {|
 # + socketTimeout - Socket timeout (in seconds) during the read/write operations with mysql server,
 #                            0 means no socket timeout
 public type Options record {|
-    SSLConfig? ssl = {};
+    SecureSocket? ssl = {};
```

Comment:
Can't we make this optional?
```suggestion
    SecureSocket ssl?;
```

Reply:
made Optional in https://github.com/ballerina-platform/module-ballerinax-mysql/pull/96/commits/45b458de1d5df35268be9b467362e48c0d0fcc17